src/redux/rootSaga.js

```diff
@@ -1,6 +1,6 @@
 import { all } from 'redux-saga/effects';
-import watchNewsLoad from './news/news.sagas';
+import getNews from './news/news.sagas';
 
 export default function* rootSaga() {
-  yield all([watchNewsLoad()]);
+  yield all([getNews()]);
```

Comment:
Rename this saga from `getNews` to `newsSaga` because it is a general saga related to `news`

Reply:
renamed